src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx

```diff
@@ -15,29 +15,23 @@ You can collect metrics from any [software that exposes Prometheus metrics](http
 
 Complete the steps below to collect Prometheus metrics.
 
-## Step 1: Sign up for your free account if you haven't already [#signup]
+<Steps>
+    <Step>
+    ## Ensure you have the prerequisites 
```

Comment:
Alternative title maybe? "Get set up" "Start here" -- not sure, but 'ensure you have the prereqs' seems a bit silly